vlib/time/README.md

```diff
@@ -1,17 +1,117 @@
+# time
+
+A time library
+
+## Features
+
+TODO
+
 ## Description:
 
 `time` provides utilities for working with time and dates:
+
 - parsing of time values expressed in one of the commonly used standard time/date formats
 - formatting of time values
 - arithmetic over times/durations
 - converting between local time and UTC (timezone support)
 - stop watches for accurately measuring time durations
 - sleeping for a period of time
 
-## Examples:
+**Example:**
+time can be comparable
+
+see: [V Playground](https://play.vlang.io/p/c121a6dda7)
 
 ```v
 import time
 
 println(time.now())
 ```
+
+### format
+
+**Example:**
+
+see: [V Playground](https://play.vlang.io/p/133d1a0ce5)
+
+```v
+import time
+
+const (
+	time_to_test = time.Time{
+		year: 1980
+		month: 7
+		day: 11
+		hour: 21
+		minute: 23
+		second: 42
+		microsecond: 123456
+		unix: 332198622
+	}
+)
+
+println(time_to_test.format())
+
+assert '1980-07-11 21:23' == time_to_test.format()
+assert '1980-07-11 21:23:42'== time_to_test.format_ss()
+assert '1980-07-11 21:23:42.123' == time_to_test.format_ss_milli()
+assert '1980-07-11 21:23:42.123456' == time_to_test.format_ss_micro()
+```
+
+**v doc:**
```

Comment:
what is the purpose of these mini sections?

Reply:
It's showing the function signature that `v doc` would show.  Might be better to have links to `modules.vlang.io` instead, so the doc comments would be available as well.

Reply:
> what is the purpose of these **v doc:** mini sections?

1 See result and parameter typing of similar functions
2 Popularize the use of `v doc` tool

Reply:
> It's showing the function signature that `v doc` would show. Might be better to have links to `modules.vlang.io` instead, so the doc comments would be available as well.

I can do that too. But I would like to keep a mini "v doc" showing similar functions.

So that the reader has the option to see everything he needs on a single page in a more minimalistic way

Reply:
It is going to get shown inside https://modules.vlang.io/time.html and that already does list all the functions/methods in the module on the right.